Cargo.toml

```diff
@@ -11,25 +11,26 @@ clap = { version = "4.1", features = ["derive"] }
 env_logger = "0.10"
 itertools = "0.10"
 log = "0.4"
-ouroboros = "0.15"
 pretty_assertions = "1.3"
+regex = "1.7.1"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 tempfile = "3.4.0"
 test-log = "0.2"
-tree-sitter = "0.20"
+tree-sitter-facade = { git = "https://github.com/tweag/tree-sitter-facade" }
```

Comment:
I don't think so. They are potentially useful outside of Topiary, for anyone who needs to target Tree-sitter both natively and through Wasm. We are clearly not the only ones who have run into that need.

And since https://github.com/silvanshade/tree-sitter-facade is archived, we are the de-facto maintainers of that code now.